imports/api/setup.app-tests.js

```diff
@@ -0,0 +1,51 @@
+import { Meteor } from 'meteor/meteor';
```

Comment:
maybe a more descriptive name to this file would be `fixtures` or `fixture-methods`, or 'test-data', as the file doesn't really do any 'setup' in that sense, but populates the database with test data